packages/client/hmi-client/src/components/model-template/tera-model-template-editor.vue

```diff
@@ -240,6 +332,42 @@ function onDrop(event) {
 	newModelTemplate.value = null;
 }
 
+const updatePosition = ({ x, y }, node: any) => {
+	if (!isMouseOverCanvas) return;
+
+	const isJunction = node.edges !== undefined;
```

Comment:
Should add a comment or fixme to explain, cause it is a bit misleading on first reading - cause it isn't clear you got two different types of nodes.